src/firefly/js/charts/ui/options/ScatterOptions.jsx

```diff
@@ -119,7 +119,7 @@ export function fieldReducer({chartId, activeTrace}) {
             fieldKey: errorMinusFieldKey(activeTrace, 'x'),
             value: get(tablesourceMappings, ['error_x.arrayminus'], ''),
             //tooltip: 'X error',
-            label: 'X Err\u2193:',
+            label: 'Error\u2193:',
```

Comment:
We might want to remove tooltip and label properties from error and errorMinus fields here - they have no effect here and are actually defined in the Errors' render (as props to ColumnOrExpression).